### ISSUE TITLE
During Upgrade provide more details that it's pulling image

### ISSUE THREAD

**Issue by deniseschannon** [rancher/os#586]
When staging an image:

`sudo ros os upgrade -s -i rancher/os:v0.4.0-rc7`

Only outputs are
```
INFO[0000] [0/1] [os-upgrade]: Starting
INFO[0032] [1/1] [os-upgrade]: Started
```

It gives no indication that it's pulling down the image.

**Comment by deniseschannon** [comment]
This is probably a duplicate of https://github.com/rancher/os/issues/488 but a specific use case.

**Comment by deniseschannon** [comment]
Closed in favor of #488
